cli/run.go

```diff
@@ -322,10 +333,8 @@ func (r WorkflowRunner) runWorkflows(tracker analytics.Tracker) (models.BuildRun
 	for i, workflowRunPlan := range plan.ExecutionPlan {
 		isLastWorkflow := i == len(plan.ExecutionPlan)-1
 		workflowToRun := r.config.Config.Workflows[workflowRunPlan.WorkflowID]
-		if workflowToRun.Title == "" {
```

Comment:
Determining the workflow title is moved to the `createWorkflowRunPlan` function.